jx-docker-build.sh

```diff
@@ -24,10 +24,10 @@ TAG=$TAG_NUM
 
 export DOCKER_REGISTRY=docker.io
 
-echo "Building ${DOCKER_REGISTRY}/${ORG}/jenkins-filerunner:${TAG}"
-docker build -t ${DOCKER_REGISTRY}/${ORG}/jenkins-filerunner:${TAG} -f Dockerfile.filerunner .
-head -n 1 Dockerfile.filerunner
-echo "Built ${DOCKER_REGISTRY}/${ORG}/jenkins-filerunner:${TAG}"
+export JENKINSFILE_RUNNER_TAG="${DOCKER_REGISTRY}/${ORG}/jenkins-filerunner:${TAG}"
+echo "Building ${JENKINSFILE_RUNNER_TAG}"
+make clean buildLocally
```

Comment:
`buildInDocker` fails on the CI instance with a strange message, cannot reproduce locally.
Maybe we could start from this impl

Reply:
Is it because docker is running in another docker container?

Reply:
Yes, I believe so. Volumes are not passed correctly.